addons/website/views/snippets/s_pricelist_cafe.xml

```diff
@@ -0,0 +1,138 @@
+                        </t>
+                        <t t-call="website.s_pricelist_cafe_item">
+                            <t t-set="name">Jasmine Green Tea</t>
+                            <t t-set="price">$3.00</t>
+                            <t t-set="description">Delicate green tea scented with jasmine blossoms, providing a soothing and floral experience.</t>
+                        </t>
+                        <t t-call="website.s_pricelist_cafe_item">
+                            <t t-set="name">Chamomile Tea</t>
+                            <t t-set="price">$4.00</t>
+                            <t t-set="description">Herbal tea made from dried chamomile flowers, known for its calming properties and gentle, apple-like flavor.</t>
+                        </t>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="s_pricelist_cafe_item">
+    <li class="s_pricelist_cafe_item mb-4" data-name="Product">
+        <p class="s_pricelist_cafe_item_title d-flex align-items-baseline fw-bold">
+            <span class="d-flex flex-grow-1 align-items-center">
+                <span t-out="name" class="s_pricelist_cafe_item_name"/>
+                <span class="s_pricelist_cafe_item_line flex-grow-1 ms-2 border-top"/>
+            </span>
+            <span t-out="price" class="s_pricelist_cafe_item_price ms-auto ps-3"/>
+        </p>
+        <p t-if="description" t-out="description" class="s_pricelist_cafe_item_description d-block mt-2 pe-5 text-muted o_small"/>
+    </li>
+</template>
+
+<template id="s_pricelist_cafe_add_product_widget">
+    <we-row string="Product">
+        <we-button data-add-item="" data-item=".s_pricelist_cafe_item:last" data-select-item="" t-att-data-apply-to="apply_to" data-no-preview="true" class="o_we_bg_brand_primary">
+            Add Product
+        </we-button>
+    </we-row>
+</template>
+
+<template id="s_pricelist_cafe_options" inherit_id="website.snippet_options">
+    <!--  Box Vertical Alignment -->
+    <xpath expr="//div[@data-js='WebsiteAnimate']" position="after">
+        <div data-selector=".s_pricelist_cafe_col">
+            <we-button-group string="Vert. Alignment" title="Vertical Alignment">
+                <we-button title="Align Top"
+                        data-select-class="justify-content-start"
+                        data-img="/website/static/src/img/snippets_options/align_solo_top.svg"/>
+                <we-button title="Align Middle"
+                        data-select-class="justify-content-center"
+                        data-img="/website/static/src/img/snippets_options/align_solo_middle.svg"/>
+                <we-button title="Align Bottom"
+                        data-select-class="justify-content-end"
+                        data-img="/website/static/src/img/snippets_options/align_solo_bottom.svg"/>
+            </we-button-group>
+        </div>
+    </xpath>
```

Comment:
@Brieuc-brd When I test these options appear twice : 
![image](https://github.com/user-attachments/assets/6ad6dfb2-3581-4fc3-b69e-5a75c266913c)

Reply:
It seems this is the case only after rebase - additionally it does not work anymore, while it did before the rebase.

Reply:
This seems to be a bug introduced in https://github.com/odoo/odoo/pull/176631 😬 
I've made a fix : https://github.com/odoo/odoo/pull/179098
@SergeBayet since you reviewed the initial PR, could you have a look at my fix ? :)